packages/contracts/utils/testHelpers.js

```diff
@@ -687,6 +687,9 @@ class TestHelper {
     if (ICR === undefined) ICR = this.toBN(this.dec(15, 17)) // 150%
     else if (typeof ICR == 'string') ICR = this.toBN(ICR)
     const assetAmount = ('value' in extraParams) ? extraParams.value : ICR.mul(totalDebt).div(price);
+    if (contracts.erc20.address == this.ZERO_ADDRESS) {
```

Comment:
Wouldn't be better to use strict equality validation: '===' instead of '==', considering that the latter does type conversion in the process, while the first don't (requiring to have same type and value)?

Reply:
✔️